lemarche/www/siaes/views.py

```diff
@@ -264,12 +266,19 @@ def form_valid(self, form: ContactForm, siae: Siae):
         )
         return HttpResponseRedirect(self.get_success_url())
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs["is_authenticated"] = self.request.user.is_authenticated
```

Comment:
```suggestion
        kwargs["user"] = self.request.user
```
pour faire comme dans `TenderCreateStepContactForm` ?